package.json

```diff
@@ -77,21 +77,24 @@
       "packageCategory": "games",
       "depends": [
         "gawk",
-        "curl"
```

Comment:
maybe add tar as dependency, too?

Reply:
I think its impossible for a distro do not have TAR installed since its used to extract basically everything. If this was rar or zip, perhaps we would need unzip or unrar but not tar.

Reply:
Yes i agree but then curl should also not be on the list :D. I mean it does not hurt if with add tar to the list ? If someone does not have it for some reason he get an error message, else not.

Reply:
Its complicated to have dependencies here because this will ask for installing when installing the package.
some time ago I added wine as a dependency here and it broke several installations because some distros have different name for wine. So this is last resource.
I will keep as it is for now and lets see if someone will appear with a missing tar some day.